types/protocol/filter.go

```diff
@@ -0,0 +1,42 @@
+package protocol
+
+type FilterOperator int
+
+const (
+	Equal FilterOperator = iota
+	NotEqual
+	Lesser
+	LesserEqual
+	Greater
+	GreaterEqual
+)
+
+type Filter struct {
+	Field    string
+	Operator FilterOperator
+	Value    []interface{}
+}
+
+func EqualFilter(field string, vals []interface{}) Filter {
```

Comment:
are these funcs **really** required?

Reply:
They're for dev experience, @AsafEitani requested them as helpers.

Reply:
I did 😄